src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -423,6 +427,11 @@ public void run() {
    //                           Private methods
    // ***********************************************************************
 
+   private void closeConnection(final PoolEntry poolEntry, final String closureReason)
+   {
+	   closeConnection(poolEntry, closureReason, true);
```

Comment:
Kill the third parameter to ``closeConnection()``, we cannot skip calling ``close()``.  The "penalty" in the rare case where an exception *is* thrown by ``close()`` is merely an additional log (at debug level!).